### ISSUE TITLE
Hello, harap baca saya!

### ISSUE THREAD

**Issue by indrakaw** [GitIndonesia/handbook#1]
**Tentang repo:**
Repo ini berisikan dokumen dan panduan git, diterutamakan kepada pemula. Disajikan oleh @GitIndonesia. 

**Orientasi:**
Menggunakan Bahasa Indonesia. Karena sebagian dari warga negara Indonesia belum mengerti bahasa inggris.

**Penulisan:** 
Bermodul.

**Catatan/tambahan:**

- Ada repo yang membahas git, terjemahan Bahasa Indonesia: [progit-id](https://github.com/progit/progit2-id)

**Comment by dikiaap** [comment]
@indrakaw bagus, lanjutkan.

**Comment by troke12** [comment]
lebih enak pake wiki aja gan (?)

**Comment by indrakaw** [comment]
@troke12: no. Karena kalau pakai wiki, tidak fokus ke main repo. Saya ingin repo ini menjadi `GitIndonesia/handbook`, bukan `GitIndonesia/hanbook/wiki`.
